components/eventing-controller/pkg/backend/nats/jetstream/jetstream.go

```diff
@@ -651,7 +651,6 @@ func (js *JetStream) getDefaultSubscriptionOptions(consumer SubscriptionSubjectI
 		toJetStreamConsumerDeliverPolicyOptOrDefault(js.Config.JSConsumerDeliverPolicy),
 		nats.MaxAckPending(subConfig.MaxInFlightMessages),
 		nats.MaxDeliver(jsConsumerMaxRedeliver),
-		nats.AckWait(jsConsumerAcKWait),
```

Comment:
what do you think should we not keep this in case the call `msg.NakWithDelay(jsConsumerNakDelay)` fails?
if the nack function call fails for any reason, it will not be redelivered anymore.